README.md

```diff
@@ -0,0 +1,116 @@
+# Escoria Rewrite
+
+Libre framework for the creation of point-and-click adventure games with
+the MIT-licensed multi-platform game engine [Godot Engine](https://godotengine.org).
+
+It is designed so that you can claim it for yourself and modify it to match
+the needs of your specific game and team.
+
+This repository is big rewrite of the original [Escoria framework](https://github.com/godotengine/escoria/tree/master). Its purpose is to make Escoria work as a plugin for the Godot Engine editor, instead of being a collection of scripts and scenes. It is intended to be easier to use and easier to maintain. 
+
+If you're encountering issues or incompatibilities, please raise an issue on [Escoria's Github repository](https://github.com/godotengine/escoria/issues).
+
+## History
+
+This framework was initially developed for the adventure game
+[The Interactive Adventures of Dog Mendonça and Pizzaboy®](http://store.steampowered.com/app/330420)
+and later streamlined for broader usages and open sourced as promised
+to the backers of the Dog Mendonça Kickstarter campaign.
+
+## Authors
+
+In alphabetical order:
+
+* ArturM
+* Sylvain Beucler - beuc
+* Fleskevor
+* Ariel Manzur - punto (original author)
+* Julian Murgia - @StraToN
+* Dennis Ploeger - @dploeger
+* Markus Törnqvist - mjtorn
+
+## Documentation
+
+* Getting started
+* [Architecture](docs/architecture.md)
+* [Configuration](docs/configuration.md)
+* [ESC language documentation](api/esc.md)
+
+## Roadmap
+
+The rewrite is currently ongoing and certain features of Escoria are missing or require optimization:
+
+### Basic features
+
+* [ ] Implement on sound volume management
```

Comment:
This was added in 5d1df7aaa98ecccab62818ab9dfe866d40bd3214.

Reply:
No, I think I meant the volume management in the options which you did. Removing that then. 👍🏽